modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operators-from-a-cluster-using-web-console-{context}']
+= Deleting Operators from a cluster using the web console
+
+To delete an installed Operator from the selected namespace through the Web console, follow these steps:
+
+.Procedure
+
+. Navigate in the web console to the *Catalog → Operator Management* page.
+. Select the *Operator Subscriptions* tab to see the list of installed Operators
+for the namespace.
+. Find the Operator you want to delete (in this example, jaeger) and click
+the three-dot drop-down box in far right column.
++
+image::olm-operator-delete.png[]
+
+. Select *Remove Subscription*.
+. When prompted from the *Remove Subscription* pop-up, optionally select the
+`Also completely remove the jaeger Operator from the selected namespace`
+checkbox, if you want all components related to the installation to be removed.
+This removes the CSV, which in turn removes the pods, the deployments, the
```

Comment:
Remove " This action trickles down and releases everything."

Reply:
Awwwww. Those were the exact words of the developer. I thought it was almost poetic. Well, so much for poetry today.